Carthage Sample/Carthage-sample-app.xcodeproj/project.pbxproj

```diff
@@ -434,7 +450,7 @@
 				DEVELOPMENT_TEAM = 68F5DK9B95;
 				FRAMEWORK_SEARCH_PATHS = "$(inherited)";
 				INFOPLIST_FILE = "Carthage-sample-app/Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 15.4;
```

Comment:
Why change this?

Reply:
I added it to use the async methods, but I reverted it and added some conditionals based on the available iOS